# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Arpad Biro <biro_arpad@yahoo.com>, 2006-2007
# Dimitris Glezos <glezos@indifex.com>, 2011
# Nikolas Slivka <snicore@gmail.com>, 2010
# Peter Borsa <peter.borsa@gmail.com>, 2013
# Ferenc Teknős <teknos.ferenc@gmail.com>, 2013
# Zoltan Hoppár <hopparz@gmail.com>, 2012-2013
# Meskó Balázs <meskobalazs@gmail.com>, 2018. #zanata
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2018-08-03 10:39+0200\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"PO-Revision-Date: 2018-03-03 11:18+0000\n"
"Last-Translator: Copied by Zanata <copied-by-zanata@zanata.org>\n"
"Language-Team: Hungarian (http://www.transifex.com/projects/p/fedora/"
"language/hu/)\n"
"Language: hu\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: Zanata 4.5.0\n"

#: ../setroubleshoot.desktop.in.h:1
msgid "SELinux Troubleshooter"
msgstr "SELinux hibakezelő"

#: ../setroubleshoot.desktop.in.h:2
msgid "Troubleshoot SELinux access denials"
msgstr "SELinux hozzáférési elutasítások kezelése"

#: ../setroubleshoot.desktop.in.h:3
msgid "policy;security;selinux;avc;permission;mac;alert;sealert;"
msgstr "házirend;biztonság;selinux;avc;jogosultság;mac;riasztás;"

#: ../src/setroubleshoot/audit_data.py:946
#, python-format
msgid "port %s"
msgstr "port: %s"

#: ../src/setroubleshoot/audit_data.py:948 ../src/setroubleshoot/browser.py:773
#: ../src/setroubleshoot/browser.py:848 ../src/setroubleshoot/rpc.py:558
#: ../src/setroubleshoot/util.py:282
msgid "Unknown"
msgstr "Ismeretlen"

#: ../src/setroubleshoot/audit_data.py:1037
#, python-format
msgid "%s \n"
"**** Invalid AVC allowed in current policy ***\n"
msgstr "%s \n"
"**** Hibás AVC engedélyezve a jelenlegi szabályzatban ***\n"

#: ../src/setroubleshoot/audit_data.py:1039
#, python-format
msgid ""
"%s \n"
"**** Invalid AVC dontaudited in current policy.  'semodule -B' will turn on "
"dontaudit rules. ***\n"
msgstr ""
"%s \n"
"**** Hibás AVC dontaudit a jelenlegi szabályzatban.  Az „semodule -B” "
"kapcsoló kapcsolja be a dontaudit szabályzatot. ***\n"

#: ../src/setroubleshoot/audit_data.py:1041
msgid "Must call policy_init first"
msgstr "A policy_init kérést kell elsőként meghívni"

#: ../src/setroubleshoot/audit_data.py:1043
#, python-format
msgid "%s \n"
"**** Invalid AVC bad target context. ***\n"
msgstr "%s \n"
"**** Hibás AVC: hibás célkörnyezet. ***\n"

#: ../src/setroubleshoot/audit_data.py:1045
#, python-format
msgid "%s \n"
"**** Invalid AVC bad source context. ***\n"
msgstr "%s \n"
"**** Hibás AVC: rossz forráskörnyezet. ***\n"

#: ../src/setroubleshoot/audit_data.py:1047
#, python-format
msgid "%s \n"
"**** Invalid AVC bad type class ***\n"
msgstr "%s \n"
"**** Hibás AVC: rossz típusosztály ***\n"

#: ../src/setroubleshoot/audit_data.py:1049
#, python-format
msgid "%s \n"
"**** Invalid AVC bad permission ***\n"
msgstr "%s \n"
"**** Hibás AVC: hibás jogosultság ***\n"

#: ../src/setroubleshoot/audit_data.py:1051
msgid "Error during access vector computation"
msgstr "Hiba a hozzáférési vektor kiszámításánál"

#: ../src/setroubleshoot/browser.py:204 ../gui/browser.glade.h:14
msgid "SELinux Alert Browser"
msgstr "SELinux riasztási böngésző"

#: ../src/setroubleshoot/browser.py:208 ../gui/browser.glade.h:21
msgid "The source process:"
msgstr "A forrásfolyamat:"

#: ../src/setroubleshoot/browser.py:213 ../gui/browser.glade.h:17
msgid "Yes"
msgstr "Igen"

#: ../src/setroubleshoot/browser.py:215 ../gui/browser.glade.h:19
msgid "No"
msgstr "Nem"

#: ../src/setroubleshoot/browser.py:220 ../gui/browser.glade.h:22
msgid "Attempted this access:"
msgstr "Ez próbált meg hozzáférni:"

#: ../src/setroubleshoot/browser.py:234 ../gui/browser.glade.h:12
msgid "SETroubleshoot Details Window"
msgstr "SETroubleshoot részletező ablak"

#: ../src/setroubleshoot/browser.py:237 ../gui/browser.glade.h:16
msgid "Would you like to receive alerts?"
msgstr "Szeretne riasztásokat kapni?"

#: ../src/setroubleshoot/browser.py:241 ../gui/browser.glade.h:26
msgid "Notify Admin"
msgstr "Rendszergazda értesítése"

#: ../src/setroubleshoot/browser.py:244 ../src/setroubleshoot/browser.py:250
#: ../gui/browser.glade.h:8
msgid "Troubleshoot"
msgstr "Hibaelhárítás"

#: ../src/setroubleshoot/browser.py:247 ../gui/browser.glade.h:28
msgid "Details"
msgstr "Részletek"

#: ../src/setroubleshoot/browser.py:254 ../gui/browser.glade.h:7
msgid "SETroubleshoot Alert List"
msgstr "SETroubleshoot riasztási lista"

#: ../src/setroubleshoot/browser.py:256 ../gui/browser.glade.h:37
msgid "List All Alerts"
msgstr "Összes riasztás felsorolása"

#: ../src/setroubleshoot/browser.py:303
msgid "#"
msgstr "#"

#: ../src/setroubleshoot/browser.py:303
msgid "Source Process"
msgstr "Forrás folyamat"

#: ../src/setroubleshoot/browser.py:303
msgid "Attempted Access"
msgstr "Hozzáférési próbálkozás"

#: ../src/setroubleshoot/browser.py:303
msgid "On this"
msgstr "Itt"

#: ../src/setroubleshoot/browser.py:303
msgid "Occurred"
msgstr "Előfordulás"

#: ../src/setroubleshoot/browser.py:303
msgid "Status"
msgstr "Állapot"

#: ../src/setroubleshoot/browser.py:377 ../src/setroubleshoot/browser.py:812
#: ../src/setroubleshoot/browser.py:856
msgid "Notify"
msgstr "Értesítés"

#: ../src/setroubleshoot/browser.py:378
msgid "Notify alert in the future."
msgstr "Riasztás a jövőben is."

#: ../src/setroubleshoot/browser.py:381 ../src/setroubleshoot/browser.py:814
#: ../src/setroubleshoot/browser.py:854 ../gui/browser.glade.h:29
msgid "Ignore"
msgstr "Mellőzés"

#: ../src/setroubleshoot/browser.py:382 ../gui/browser.glade.h:30
msgid "Ignore alert in the future."
msgstr "Riasztás mellőzése a jövőben."

#: ../src/setroubleshoot/browser.py:419
msgid "<b>If you were trying to...</b>"
msgstr "<b>Ha ezt szeretné…</b>"

#: ../src/setroubleshoot/browser.py:428
msgid "<b>Then this is the solution.</b>"
msgstr "<b>Akkor ez a megoldás.</b>"

#: ../src/setroubleshoot/browser.py:525
msgid "Plugin\n"
"Details"
msgstr "Bővítmény\n"
"részletei"

#: ../src/setroubleshoot/browser.py:540
msgid "Report\n"
"Bug"
msgstr "Hiba\n"
"jelentése"

#: ../src/setroubleshoot/browser.py:570
#, python-format
msgid "Plugin: %s "
msgstr "Bővítmény: %s "

#: ../src/setroubleshoot/browser.py:637
msgid "Unable to grant access."
msgstr "Nem sikerült engedélyezni a hozzáférést."

#: ../src/setroubleshoot/browser.py:668 ../src/setroubleshoot/browser.py:810
#, python-format
msgid "Alert %d of %d"
msgstr "Riasztás %d / %d"

#: ../src/setroubleshoot/browser.py:793
#, python-format
msgid "On this %s:"
msgstr "Itt: %s:"

#: ../src/setroubleshoot/browser.py:841 ../src/setroubleshoot/browser.py:847
#: ../src/setroubleshoot/signature.py:422
msgid "N/A"
msgstr "–"

#: ../src/setroubleshoot/browser.py:888 ../src/setroubleshoot/browser.py:889
#: ../gui/browser.glade.h:34
msgid "No Alerts"
msgstr "Nincs riasztás"

#: ../src/setroubleshoot/browser.py:907
msgid "SELinux has detected a problem."
msgstr "Az SElinux hibát észlelt."

#: ../src/setroubleshoot/browser.py:1023
msgid "Sealert Error"
msgstr "Sealert hiba"

#: ../src/setroubleshoot/browser.py:1036
msgid "Sealert Message"
msgstr "Sealert üzenet"

#. -----------------------------------------------------------------------------
#: ../src/setroubleshoot/errcode.py:83
msgid "signature not found"
msgstr "aláírás nem található"

#: ../src/setroubleshoot/errcode.py:84
msgid "multiple signatures matched"
msgstr "több aláírás egyezik"

#: ../src/setroubleshoot/errcode.py:85
msgid "id not found"
msgstr "azonosító nem található"

#: ../src/setroubleshoot/errcode.py:86
msgid "database not found"
msgstr "adatbázis nem található"

#: ../src/setroubleshoot/errcode.py:87
msgid "item is not a member"
msgstr "az elem nem tag"

#: ../src/setroubleshoot/errcode.py:88
msgid "illegal to change user"
msgstr "nem változtathat felhasználót"

#: ../src/setroubleshoot/errcode.py:89
msgid "method not found"
msgstr "eljárás nem található"

#: ../src/setroubleshoot/errcode.py:90
msgid "cannot create GUI"
msgstr "GUI indítása sikertelen"

#: ../src/setroubleshoot/errcode.py:91
msgid "value unknown"
msgstr "ismeretlen érték"

#: ../src/setroubleshoot/errcode.py:92
msgid "cannot open file"
msgstr "fájl nem megnyitható "

#: ../src/setroubleshoot/errcode.py:93
msgid "invalid email address"
msgstr "Érvénytelen email cím"

#. gobject IO Errors
#: ../src/setroubleshoot/errcode.py:96
msgid "socket error"
msgstr "socket hiba"

#: ../src/setroubleshoot/errcode.py:97
msgid "connection has been broken"
msgstr "kapcsolat megszakadt"

#: ../src/setroubleshoot/errcode.py:98
msgid "Invalid request. The file descriptor is not open"
msgstr "Érvénytelen kérés. A fájl leíró nincs nyitva"

#: ../src/setroubleshoot/errcode.py:100
msgid "insufficient permission to modify user"
msgstr "elégtelen jogosultság a felhasználó megváltoztatására"

#: ../src/setroubleshoot/errcode.py:101
msgid "authentication failed"
msgstr "hitelesítés sikertelen"

#: ../src/setroubleshoot/errcode.py:102
msgid "user prohibited"
msgstr "felhasználó letiltva"

#: ../src/setroubleshoot/errcode.py:103
msgid "not authenticated"
msgstr "nincs azonosítva"

#: ../src/setroubleshoot/errcode.py:104
msgid "user lookup failed"
msgstr "felhasználó keresés eredménytelen"

#: ../src/setroubleshoot/gui_utils.py:56 ../src/sealert:564
#, c-format, python-format
msgid "Opps, %s hit an error!"
msgstr "Hoppá, %s hibaüzenetet kapott!"

#: ../src/setroubleshoot/gui_utils.py:58 ../gui/fail_dialog.glade.h:1
msgid "Error"
msgstr "Hiba"

#: ../src/setroubleshoot/Plugin.py:92
msgid ""
"If you want to allow $SOURCE_BASE_PATH to have $ACCESS access on the "
"$TARGET_BASE_PATH $TARGET_CLASS"
msgstr ""

#: ../src/setroubleshoot/server.py:219
#, python-format
msgid " For complete SELinux messages run: sealert -l %s"
msgstr ""

#: ../src/setroubleshoot/server.py:407
#, python-format
msgid "The user (%s) cannot modify data for (%s)"
msgstr "A (%s) felhasználó nem módosíthatja (%s) adatait"

#: ../src/setroubleshoot/server.py:479 ../src/sealert:317
msgid "Started"
msgstr "Elindítva"

#: ../src/setroubleshoot/server.py:677
msgid "AVC"
msgstr "AVC"

#: ../src/setroubleshoot/server.py:745
msgid "Audit Listener"
msgstr "Audit figyelő"

#: ../src/setroubleshoot/signature.py:89
msgid "Never Ignore"
msgstr "Soha ne mellőzze"

#: ../src/setroubleshoot/signature.py:90
msgid "Ignore Always"
msgstr "Mindig mellőzze"

#: ../src/setroubleshoot/signature.py:91
msgid "Ignore After First Alert"
msgstr "Mellőzze az első riasztás után"

#: ../src/setroubleshoot/signature.py:211
msgid "directory"
msgstr "könyvtár"

#: ../src/setroubleshoot/signature.py:212
msgid "semaphore"
msgstr "szemafor"

#: ../src/setroubleshoot/signature.py:213
msgid "shared memory"
msgstr "osztott memória"

#: ../src/setroubleshoot/signature.py:214
msgid "message queue"
msgstr "üzenetsor"

#: ../src/setroubleshoot/signature.py:215
msgid "message"
msgstr "üzenet"

#: ../src/setroubleshoot/signature.py:216
msgid "file"
msgstr "fájl"

#: ../src/setroubleshoot/signature.py:217
msgid "socket"
msgstr "port"

#: ../src/setroubleshoot/signature.py:218
msgid "process"
msgstr "folyamat"

#: ../src/setroubleshoot/signature.py:219
msgid "process2"
msgstr ""

#: ../src/setroubleshoot/signature.py:220
msgid "filesystem"
msgstr "fájlrendszer"

#: ../src/setroubleshoot/signature.py:221
msgid "node"
msgstr "csomópont"

#: ../src/setroubleshoot/signature.py:222
msgid "capability"
msgstr "funkció"

#: ../src/setroubleshoot/signature.py:223
msgid "capability2"
msgstr ""

#: ../src/setroubleshoot/signature.py:406
#, python-format
msgid "%s has a permissive type (%s). This access was not denied."
msgstr ""
"%s egy engedélyező típussal rendelkezik (%s). Ez a hozzáférés nem volt "
"megtagadva."

#: ../src/setroubleshoot/signature.py:409
msgid "SELinux is in permissive mode. This access was not denied."
msgstr "SElinux átengedő módban van. Ez a hozzáférés nem volt blokkolva."

#: ../src/setroubleshoot/signature.py:463
#, python-format
msgid "SELinux is preventing %s from using the %s access on a process."
msgstr ""
"SELinux megakadályozta %s számára %s elérhető hozzáférést a folyamaton."

#: ../src/setroubleshoot/signature.py:463
#, python-format
msgid "SELinux is preventing %s from using the '%s' accesses on a process."
msgstr ""
"SELinux megakadályozta %s számára %s elérhető hozzáféréseket a folyamaton."

#: ../src/setroubleshoot/signature.py:466
#, python-format
msgid "SELinux is preventing %s from using the %s capability."
msgstr "SELinux megakadályozta %s számára %s funkció használatát."

#: ../src/setroubleshoot/signature.py:466
#, python-format
msgid "SELinux is preventing %s from using the '%s' capabilities."
msgstr "SELinux megakadályozta %s számára '%s' funkciók használatát."

#: ../src/setroubleshoot/signature.py:468
#, python-format
msgid "SELinux is preventing %s from %s access on the %s labeled %s."
msgstr ""
"SELinux megakadályozta %s alkalmazás %s hozzáférését %s erőforrás %s "
"címkézett elemén."

#: ../src/setroubleshoot/signature.py:468
#, python-format
msgid "SELinux is preventing %s from '%s' accesses on the %s labeled %s."
msgstr ""
"SElinux megakadályozta %s számára '%s' hozzáféréseit %s címkével ezen: %s"

#: ../src/setroubleshoot/signature.py:469
#, python-format
msgid "SELinux is preventing %s from %s access on the %s %s."
msgstr "SELinux megakadályozta %s számára %s hozzáférését itt: %s %s."

#: ../src/setroubleshoot/signature.py:469
#, python-format
msgid "SELinux is preventing %s from '%s' accesses on the %s %s."
msgstr "SELinux megakadályozta %s számára '%s' hozzáféréseit itt: %s %s."

#: ../src/setroubleshoot/signature.py:514
msgid "Additional Information:\n"
msgstr "További információ:\n"

#: ../src/setroubleshoot/signature.py:515
msgid "Source Context"
msgstr "Forrás kontextus"

#: ../src/setroubleshoot/signature.py:516
msgid "Target Context"
msgstr "Cél kontextus"

#: ../src/setroubleshoot/signature.py:517
msgid "Target Objects"
msgstr "Cél objektumok"

#: ../src/setroubleshoot/signature.py:518
msgid "Source"
msgstr "Forrás"

#: ../src/setroubleshoot/signature.py:519
msgid "Source Path"
msgstr "Forrás elérési útja"

#: ../src/setroubleshoot/signature.py:520
msgid "Port"
msgstr "Port"

#: ../src/setroubleshoot/signature.py:522
#: ../src/setroubleshoot/signature.py:524
msgid "Host"
msgstr "Kiszolgáló"

#: ../src/setroubleshoot/signature.py:525
msgid "Source RPM Packages"
msgstr "Forrás RPM csomagok"

#: ../src/setroubleshoot/signature.py:526
msgid "Target RPM Packages"
msgstr "Cél RPM csomagok"

#: ../src/setroubleshoot/signature.py:527
msgid "Policy RPM"
msgstr "Szabályzat RPM"

#: ../src/setroubleshoot/signature.py:528
msgid "Selinux Enabled"
msgstr "SELinux bekapcsolva"

#: ../src/setroubleshoot/signature.py:529
msgid "Policy Type"
msgstr "Szabályzat típus"

#: ../src/setroubleshoot/signature.py:530
msgid "Enforcing Mode"
msgstr "Kényszerítő mód"

#: ../src/setroubleshoot/signature.py:532
#: ../src/setroubleshoot/signature.py:534
msgid "Host Name"
msgstr "Kiszolgáló neve"

#: ../src/setroubleshoot/signature.py:539
#: ../src/setroubleshoot/signature.py:541
msgid "Platform"
msgstr "Platform"

#: ../src/setroubleshoot/signature.py:542
msgid "Alert Count"
msgstr "Riasztás mennyisége"

#: ../src/setroubleshoot/signature.py:544
msgid "First Seen"
msgstr "Először megjelent"

#: ../src/setroubleshoot/signature.py:545
msgid "Last Seen"
msgstr "Utoljára megjelent"

#: ../src/setroubleshoot/signature.py:546
msgid "Local ID"
msgstr "Helyi azonosító"

#: ../src/setroubleshoot/signature.py:548
msgid "Raw Audit Messages"
msgstr "Nyers auditálási üzenetek"

#: ../src/setroubleshoot/signature.py:602
#, python-format
msgid "\n"
"\n"
"*****  Plugin %s (%.4s confidence) suggests   "
msgstr "\n"
"\n"
"*****  Plugin %s (%.4s bizonyossággal) javasolja a következőt   "

#: ../src/setroubleshoot/signature.py:605
msgid "*"
msgstr "*"

#: ../src/setroubleshoot/signature.py:606
#: ../src/setroubleshoot/signature.py:608
msgid "\n"
msgstr "\n"

#: ../src/setroubleshoot/signature.py:610
msgid "\n"
"Then "
msgstr "\n"
"Akkor "

#: ../src/setroubleshoot/signature.py:613
msgid "\n"
"Do\n"
msgstr "\n"
"Tedd\n"

#: ../src/setroubleshoot/signature.py:615
msgid "\n"
"\n"
msgstr "\n"
"\n"

#: ../src/updater.py:18
#, python-format
msgid "current: %s "
msgstr "jelenlegi: %s "

#: ../src/updater.py:27
#, python-format
msgid "newer: %s "
msgstr "újabb: %s"

#: ../src/seappletlegacy.c:189 ../src/seapplet:133
msgid "New SELinux security alert"
msgstr "Új SELinux biztonsági riasztás"

#: ../src/seappletlegacy.c:189 ../src/seapplet:134
msgid "AVC denial, click icon to view"
msgstr "AVC tiltás. Kattintson az ikonra, hogy megtekinthesse"

#: ../src/seappletlegacy.c:195 ../src/seappletlegacy.c:246 ../src/seapplet:137
msgid "Dismiss"
msgstr "Eldobás"

#: ../src/seappletlegacy.c:205 ../src/seappletlegacy.c:256 ../src/seapplet:138
#: ../src/seapplet:139
msgid "Show"
msgstr "Megjelenítés"

#. set tooltip
#: ../src/seappletlegacy.c:440 ../src/sealert:70
msgid "SELinux AVC denial, click to view"
msgstr "SELinux AVC megtagadás. Kattintson, hogy megtekinthesse"

#: ../src/seapplet:160
msgid "SELinux Troubleshooter: Applet requires SELinux be enabled to run"
msgstr ""

#: ../src/sealert:120
msgid "SELinux not enabled, sealert will not run on non SELinux systems"
msgstr ""
"SElinux nincs engedélyezve, sealert nem fog működni nem SElinux rendszereken"

#: ../src/sealert:170
msgid "Not fixable."
msgstr "Nem javítható."

#: ../src/sealert:177
#, c-format
msgid "Successfully ran %s"
msgstr "%s sikeresen lefutott"

#: ../src/sealert:184
#, c-format
msgid "Plugin %s not valid for %s id"
msgstr "Plugin %s nem érvényes a %s ID-hez"

#: ../src/setroubleshootd:79
msgid "SELinux not enabled, setroubleshootd exiting..."
msgstr "SELinux nincs engedélyezve, setroubleshootd kilép..."

#: ../src/setroubleshootd:111
#, c-format
msgid "fork #1 failed: %d (%s)"
msgstr "fork #1 sikertelen: %d (%s)"

#: ../gui/browser.glade.h:1
msgid ""
"Copyright (c) 2010\n"
"Thomas Liu <tliu@redhat.com>\n"
"Máirín Duffy <duffy@redhat.com>\n"
"Daniel Walsh <dwalsh@redhat.com>\n"
"John Dennis <jdennis@redhat.com>\n"
msgstr ""

#: ../gui/browser.glade.h:9
msgid "Troubleshoot selected alert"
msgstr "Hibaelhárítás a kiválasztott riasztáson"

#: ../gui/browser.glade.h:10
msgid "Delete"
msgstr ""

#: ../gui/browser.glade.h:11
msgid "Delete Selected Alerts"
msgstr "Kijelölt riasztások törlése"

#: ../gui/browser.glade.h:13
msgid "Close"
msgstr ""

#: ../gui/browser.glade.h:15
msgid "<b>SELinux has detected a problem.</b>"
msgstr "<b>SELinux hibát érzékelt.</b>"

#: ../gui/browser.glade.h:18
msgid "Turn on alert pop-ups."
msgstr "Felugró riasztási üzenetek bekapcsolása."

#: ../gui/browser.glade.h:20
msgid "Turn off alert pop-ups."
msgstr "Felugró riasztási üzenetek kikapcsolása."

#: ../gui/browser.glade.h:23
msgid "On this file:"
msgstr "Ezen a fájlon:"

#: ../gui/browser.glade.h:24
msgid "label"
msgstr "cimke"

#: ../gui/browser.glade.h:25
msgid ""
"Read alert troubleshoot information.  May require administrative privileges "
"to remedy."
msgstr ""
"Olvassa el a riasztási problémamegoldó információkat.  Lehet hogy "
"adminisztrációs jogokra lesz szüksége a megoldáshoz."

#: ../gui/browser.glade.h:27
msgid "Email alert to system administrator."
msgstr "Email risztás a rendszer adminisztátorának"

#: ../gui/browser.glade.h:31
msgid "Delete current alert from the database."
msgstr "Jelenlegi riasztás törlése az adatbázisból"

#: ../gui/browser.glade.h:32
msgid "Previous"
msgstr ""

#: ../gui/browser.glade.h:33
msgid "Show previous alert."
msgstr "Előző riasztás"

#: ../gui/browser.glade.h:35
msgid "Next"
msgstr ""

#: ../gui/browser.glade.h:36
msgid "Show next alert."
msgstr "Következő riasztás"

#: ../gui/browser.glade.h:38
msgid "List all alerts in the database."
msgstr "Az összes riasztás listázása az azdatbázisból."

#: ../gui/bug_report.glade.h:1
msgid "Review and Submit Bug Report"
msgstr "Felülvizsgálja és beküldi a hibajelentést"

#: ../gui/bug_report.glade.h:2
msgid "<span size='large' weight='bold'>Review and Submit Bug Report</span>"
msgstr ""
"<span size='large' weight='bold'>Felülvizsgálja és beküldi a hiba jelentést</"
"span>"

#: ../gui/bug_report.glade.h:3
msgid ""
"You may wish to review the error output that will be included in this bug "
"report and modify it to exclude any sensitive data below."
msgstr ""
"Előfordulhat, hogy szeretné felülvizsgálni a hiba kimenetét amit a "
"hibajelentésbe fogunk foglalni, és változtatni kíván rajta, azzal, hogy "
"eltávolít érzékeny adatokat. Alább ezt megteheti."

#: ../gui/bug_report.glade.h:4
msgid "Included error output:"
msgstr "Benne foglalt hiba kimenet:"

#: ../gui/bug_report.glade.h:5
msgid "Submit Report"
msgstr "Jelentés beküldése"

#: ../gui/fail_dialog.glade.h:2 ../gui/success_dialog.glade.h:2
msgid ""
"This operation was completed.  The quick brown fox jumped over the lazy dog."
msgstr ""
"Ez a művelet befejeződött.  A fürge vörösesbarna róka átugrotta a lusta "
"kutyát."

#: ../gui/success_dialog.glade.h:1
msgid "Success!"
msgstr "Sikerült!"

#: ../gui/success_dialog.glade.h:3
msgid "button"
msgstr "gomb"
